saleor/graphql/discount/types/promotions.py

```diff
@@ -99,3 +115,65 @@ def resolve_promotion(root: models.PromotionRule, info: ResolveInfo):
     @staticmethod
     def resolve_channels(root: models.PromotionRule, info: ResolveInfo):
         return ChannelsByPromotionRuleIdLoader(info.context).load(root.id)
+
+
+class PromotionEvent(ModelObjectType[models.PromotionEvent]):
+    id = graphene.GlobalID()
+    date = graphene.DateTime(description="Date when event happened.")
+    type = PromotionEventsEnum(description="Promotion event type.")
+    user = graphene.Field(User, description="User who performed the action.")
```

Comment:
The field contains additional permissions, we should mention it in the description.
Have you considered using `PermissionsField` here? It could help with handling the perms and extending the description